### PR TITLE
Product page Filters

### PR DESCRIPTION
**Pull request by sakshamAK**
- Completed Filters in product page...
 - Filters include:
 - Sort price high to low and low to high
 - Sort by range
 - include out oF stock
 - show Fast delivery items
 - show on sale items
 - show items by categories:
1.   Paper Cuttings
1.   Shadow Box
2.  Quilling
3.  Layering Paper Art
4.  Origami
5.  Kirigami
6.  3D Origami
7.  Miniatures

- Show items by ratings
- Show items by mount types
- Show items by colors 
image
image
(recording got zoomed in dont know why)

https://user-images.githubusercontent.com/55906931/159635185-bdf40317-9b00-4567-8b9e-0a2f0ba6c3ff.mp4

### CODE REVIEW

**Comment by techieshubham30** [comment]
code looks good to me, but I think you have raised pr to merge into master.

**Comment by sakshamAK** [comment]
> code looks good to me, but I think you have raised pr to merge into master.

Yep I accidently created all my branches from master. Planning to rename it to dev.

File: src/pages/Components/Navigation/Navigation.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+body {
+    font-family:  Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+.nav-header {
+    background-color: #ffffff80;
```
Comment (mimansha-swarup) [inline src/pages/Components/Navigation/Navigation.css:6]: using variables will be great

**Review by mimansha-swarup — APPROVED** [review]
everything is good
just use variables in CSS for colors and other stuff

File: src/pages/Components/Navigation/Navigation.css (hunk 1 above)
Comment (sakshamAK) [reply src/pages/Components/Navigation/Navigation.css:6]: sure thanks.

**Pull request merged**